visual-tests/layout/rectangles.js

```diff
@@ -149,13 +166,42 @@
     var yScale = d3.scale.linear()
         .range([height, 0]);
 
-    var smart = fc.layout.rectangles(strategy())
-        .xScale(xScale)
-        .yScale(yScale)
-        .size([itemWidth, itemHeight])
-        .component(label);
+    function useStrategy(strategyToUse) {
```

Comment:
I like the update to the visual tests. Can you please delete all the un-used junk at the top of this file too?